addons/project/models/project.py

```diff
@@ -2236,6 +2236,7 @@ def _task_message_auto_subscribe_notify(self, users_per_task):
                     record_name=task.display_name,
                     email_layout_xmlid='mail.mail_notification_layout',
                     model_description=task_model_description,
+                    mail_auto_delete=False,
```

Comment:
Why this change ?

Reply:
Because of the test  `test_personal_stage_search`
Looks like `personal_stage_id` expect the mail to exists in the database (it was the case before, because the CRON didn't run yet)
See: https://runbot.odoo.com/runbot/build/23750491

Reply:
Strange but ok, we won't solve that here :D